lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -20,15 +20,37 @@
 
                 className: 'program-card',
 
+                attributes: function() {
+                    return {
+                        'aria-role': 'group',
+                        'aria-labelledby': 'program-' + this.model.get('id')
+                    };
+                },
+
                 tpl: _.template(programCardTpl),
 
-                initialize: function() {
+                initialize: function(data) {
+                    this.progressCollection = data.context.progressCollection; 
+                    if ( this.progressCollection ) {                   
+                        this.progressModel = this.progressCollection.findWhere({
+                            programId: this.model.get('id')
+                        });
+                    }
                     this.render();
                 },
 
                 render: function() {
-                    var templated = this.tpl(this.model.toJSON());
-                    this.$el.html(templated);
+                    var orgList = _.map(this.model.get('organizations'), function(org) {
+                            return gettext(org.key);
+                        }),
+                        data = $.extend(
+                            {},
+                            this.model.toJSON(),
+                            this.getProgramProgress(),
+                            {orgList: orgList.join(' ')}
```

Comment:
It looks like this is meant to handle programs with multiple organizations by returning a whitespace-delimited string? Any reason the separator is two spaces?